web/tests/Web.Integration.Tests/BenchmarkingWebAppWebAppClient.cs

```diff
@@ -558,13 +562,27 @@ public BenchmarkingWebAppClient SetupLocalAuthoritiesWithException()
         return this;
     }
 
-    public BenchmarkingWebAppClient SetupHighNeedsHistory(History<LocalAuthorityHighNeedsYear> history)
+    public BenchmarkingWebAppClient SetupHighNeedsHistory(App.Domain.LocalAuthorities.History<LocalAuthorityHighNeedsYear> history)
```

Comment:
It might be worth renaming this and the other `History` classes to be more specific to their purpose rather than fully-qualifying name, even just for the benefit of the tests for the time being